### ISSUE TITLE
How to reproduce the result?

### ISSUE THREAD

**Issue by Dragon-lf** [Mehran-Soltani/ChannelNet#7]
Dear Sir ,
Great Work. Please can you kindly share the full code of this paper , I will be very grateful for you.
Best Regards always ,

**Comment by YuJX7** [comment]
Indeed, it is excellent work. But this code is incomplete, and I can't even reproduce it. Have you found a way to reproduce the result? @Dragon-lf

**Comment by MartinLiuJianCheng** [comment]
I have qq number: 2285830341, if anyone want to discuss ChannelNet, please contact me, we have qq group as well.
